tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -334,4 +334,465 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTest)
     EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 3.87); // 13.20 IPLV
 
 }
+
+TEST_F(EnergyPlusFixture, SingleSpeedCoolingCoil_SEERValueTest)
```

Comment:
Nice unit tests. Please add unit tests that use the new DX coil.

Reply:
SEER value calculation is currently supported only in single speed and multi-speed DX coils. Which new cooling coil are you referring to? @mjwitte

Reply:
Coil:Cooling:DX, but I see now that standard ratings didn't make it into v9.3 for this coil type.
@rraustad @Myoldmopar We should probably hold this PR until the latest DX coil updates drop in here?

Reply:
@mjwitte Oh, okay. I have not been following the new DX coils features and how different they are. Any ways I will push all the changes I have made based on your review comment.